src/sections/Planning/Planning.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PlanningCard from './PlanningCard';
+var planningOneImage = require("../../assets/planning-1.png");
```

Comment:
nit: move the image imports to constants. and map over the constants to render PlanningCards

Reply:
Done

Reply:
Also added the map function for offering section too